storage/innobase/buf/buf0flu.cc

```diff
@@ -2152,6 +2194,8 @@ static os_thread_ret_t DECLARE_THREAD(buf_flush_page_cleaner)(void*)
       buf_flush_sync_lsn= 0;
 
     buf_pool.page_cleaner_set_idle(false);
+    if (idle_flush)
+      buf_pool.update_last_activity_count();
```

Comment:
As far as I can tell, the `buf_pool.last_activity_count` could be updated a little earlier, in the only code path where we assigned `idle_flush`. There are no time-consuming steps in between.

Reply:
done